### PR TITLE
[Feature] added local version feature for ext_deps

### PR DESCRIPTION
**Pull request by matthewfcarlson**
This allows a developer to force the SDE to accept a local version.

### CODE REVIEW

File: edk2toolext/environment/version_aggregator.py [hunk 1]
```diff
@@ -15,6 +15,9 @@
 
 
 class version_aggregator(object):
+
+    LOCAL_VERSION = "LOCAL"
```
Comment (spbrogan) [inline edk2toolext/environment/version_aggregator.py]: i don't think this belongs here.  LOCAL is a feature of external_dependency and not version aggregator.

Comment (corthon) [reply edk2toolext/environment/version_aggregator.py]: Agreed. Perhaps instead of Version == LOCAL, it could be Type == LOCAL? (Or Source?)

**Comment by corthon** [comment]
Any thoughts on this one? If it's not ready or we need to pivot, should we close it and come back later?

**Comment by matthewfcarlson** [comment]
There was discussion on whether this was a feature we event wanted to support

**Comment by spbrogan** [comment]
Lets close this for now.  Feel free to open again if the use case gains priority

**Pull request closed without merging**